packages/presentation/lib/extensions/bloc.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:presentation/bloc/base/bloc.dart';
+
+extension ConvertApiRuntime on Bloc {
+  String? formatApiRuntime(int? runtime) {
```

Comment:
why not on int?)

Reply:
I thought it would be better to add to bloc as it is related more to business logic.

Reply:
don't do it via extension. convert it in mapper of details bloc

Reply:
still not the ideal solution). look, we should have the following flow:

fetching data -> mapping it (in domain) (1) -> mapping it (in bloc) (2) (optional) -> send into ui

at the "mapping it (in bloc)" level (in our case) we should format api runtime

while now we have a situation like this

fetching data -> any mapping -> send to ui -> format runtime in bloc

so, we should have a more straightforward flow